service/src/main/resources/logback.xml

```diff
@@ -11,4 +11,5 @@
     <logger name="io.netty" level="INFO"/>
     <logger name="org.mongodb.driver" level="INFO" />
     <logger name="io.ktor" level="DEBUG" />
+    <logger name="dev.kord" level="TRACE" />
```

Comment:
I don't think this does anything. Trace is the root level.

Reply:
As you said. But I thought we might like to see it explicitly for libs.